src/openforms/forms/models/form_authentication_backend.py

```diff
@@ -0,0 +1,40 @@
+from django.db import models
+from django.utils.translation import gettext_lazy as _
+
+from openforms.authentication.registry import register as auth_register
+
+from ...authentication.fields import (
+    BackendChoiceField as AuthenticationBackendChoiceField,
+)
+from . import Form
+
+
+class FormAuthenticationBackend(models.Model):
+    form = models.ForeignKey(
+        Form,
+        on_delete=models.CASCADE,
+        related_name="authentication_backends_configuration",
+    )
+    backend = AuthenticationBackendChoiceField(_("authentication backend"))
+    options = models.JSONField(
+        _("authentication backend options"),
+        default=dict,
+        blank=True,
+    )
+
+    class Meta:
+        unique_together = ("form", "backend")
+        verbose_name = _("authentication backend")
+        verbose_name_plural = _("authentication backends")
+
+    def __str__(self):
+        return _("{backend} on {form}").format(backend=self.backend, form=self.form)
+
+    def get_backend_display(self):
+        choices = dict(auth_register.get_choices())
+        return choices.get(
+            self.backend,
+            "-",
+        )
+
+    get_backend_display.short_description = _("authentication backend")
```

Comment:
can we remove this and if we still need it, move it into the `admin.py` instead? The type checker does not like this and it's really only useful for the admin anyway.